.env.example

```diff
@@ -7,7 +7,10 @@ SESSION_SECRET="super-duper-s3cret"
 # https://letsencrypt.org/docs/expiration-emails/
 LETS_ENCRYPT_ACCOUNT_EMAIL="nx@senecacollege.ca"
 
-PORT=8080
-
 # One of: trace, debug, info, warn, error, fatal, silent
 LOG_LEVEL=debug
+
+PORT=8080
```

Comment:
@shawnyu5, it's here.

Reply:
Oh my bad

Reply:
Accidentally deleted it but added it back in one of the commits.